.github/workflows/ci.yml

```diff
@@ -6,11 +6,11 @@ on:
       - main
   push:
     branches:
-      - main
+      - IDEM-172
 
 jobs:
   build-and-lint:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
```

Comment:
I also presume if this runs fine we should update our standard to 20

Reply:
Yep, agreed.